index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./css/style.css">
+  <title>Tribute Project</title>
+</head>
+
+<body>
+  <div id="main">
+    <div class="title text-align-center">
+      <h1>Dr. Norman Borlaug</h1>
+    </div>
+
+    <div class="img-div">
+
+      <img
+        src="https://assets-global.website-files.com/5dbb30f00775d4c32191a4df/5e31d5dc2639f85bc70363bd_Microverse-homepge-500w-tab-tiny-bq.png"
+        alt="" id="image" />
+      <div class="main-content text-align-center">
+        <p class="img-caption"> Microverse is a good place to learn</p>
+        <p class="tribute-info">
+          Learning to code on your own can be lonely and difficult. At
+          Microverse, you’ll join one of the most collaborative and social
+          learning communities in the world. Every day, you, your coding
+          partner, and your stand-up team will support and hold each other
+          accountable to produce great work while remote pair programming.
+        </p>
+        <a href="https://www.microverse.org/" target="_blank" rel="noopener noreferrer" class="tribute-link">Tribute to
+          Microverse</a>
+      </div>
+    </div>
```

Comment:
Also, you can wrap this part of your code in a `section` or `article` tag to make it more semantic as it also boost the page  Search engine optimization([SEO](https://developer.mozilla.org/en-US/docs/Glossary/SEO))

Reply:
- [ ] I think we might tackle this request a bit so it is more politely and friendly stated, please refer back to my required suggestions above.